apps/src/templates/levelSummary/SummaryResponses.jsx

```diff
@@ -49,13 +52,17 @@ const SummaryResponses = ({
   const logEvent = useCallback(
     eventName => {
       const {level} = scriptData;
-      analyticsReporter.sendEvent(eventName, {
-        levelId: level.id,
-        levelName: level.name,
-        levelType: level.type,
-        sectionSelected: !!selectedSection,
-        ...scriptData.reportingData,
-      });
+      analyticsReporter.sendEvent(
+        eventName,
+        {
+          levelId: level.id,
+          levelName: level.name,
+          levelType: level.type,
+          sectionSelected: !!selectedSection,
+          ...scriptData.reportingData,
+        },
+        PLATFORMS.BOTH
```

Comment:
Add this via Bryan's request.  Hoping it is ok to add now for all events in this component... @dju90